hudi-client/hudi-client-common/src/main/java/org/apache/hudi/index/bloom/HoodieBloomIndex.java

```diff
@@ -214,12 +214,16 @@ protected List<Pair<String, BloomIndexFileInfo>> loadColumnRangesFromMetaIndex(
 
     String keyField = hoodieTable.getMetaClient().getTableConfig().getRecordKeyFieldProp();
 
+    List<Pair<String, HoodieBaseFile>> baseFilesForAllPartitions = HoodieIndexUtils.getLatestBaseFilesForAllPartitions(partitions, context, hoodieTable);
```

Comment:
this was the line where I was pointing out bringing in more to driver compared to before.

Reply:
Line 219 in the original, same question about whether this is really adding more memory usage. Can you elaborate on this?

Reply:
my bad. I thought previously we just get strings to driver and not entire HoodieBaseFile. looks like we are getting HoodieBaseFile to driver even before. we are good